app/javascript/app/components/ndcs/ndcs-enhancements-viz/ndcs-enhancements-tooltip/ndcs-enhancements-tooltip-styles.scss

```diff
@@ -0,0 +1,7 @@
+@import '~styles/settings.scss';
+
+.tooltipContainer {
```

Comment:
Updated. Sorry for the sloppy spacing issues. I had been bypassing linting in my commits because the linters previously had a lot of issues with the existing code this component is based on, but I'll make sure they lint going forward.